koekalenteri-backend/src/utils/metrics.ts

```diff
@@ -0,0 +1,17 @@
+import { APIGatewayEventRequestContextWithAuthorizer } from "aws-lambda";
+const {metricScope, Unit} = require("aws-embedded-metrics");
+
+const metricsCount = (metrics: any, label: string, count: number, context: APIGatewayEventRequestContextWithAuthorizer<any>, service: string) => { 
+    metrics.setNamespace('KoekalenteriApp');
+    metrics.putDimensions({ Service: service});
+    metrics.setProperty('RequestId', context.requestId);
+    metrics.putMetric(label, count, Unit.Count);
+}
+
+export const metricsSuccess = (metrics: any, context: APIGatewayEventRequestContextWithAuthorizer<any>, service: string) => {
+    metricsCount(metrics, "Success", 1, context, service);
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/koekalenteri/koekalenteri/pullRequest?prid=7376473)
```suggestion
    metricsCount(metrics, 'Success', 1, context, service);
```